ui/src/app/components/runtime-status-icon.tsx

```diff
@@ -0,0 +1,151 @@
+import * as React from 'react';
+import * as fp from 'lodash/fp';
+import { faCircle } from '@fortawesome/free-solid-svg-icons/faCircle';
+import { faLock } from '@fortawesome/free-solid-svg-icons/faLock';
+import { faSyncAlt } from '@fortawesome/free-solid-svg-icons/faSyncAlt';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import { RuntimeStatus } from 'generated/fetch';
+
+import colors from 'app/styles/colors';
+import { reactStyles } from 'app/utils';
+import { ComputeSecuritySuspendedError } from 'app/utils/runtime-utils';
+import {
+  CompoundRuntimeOpStore,
+  compoundRuntimeOpStore,
+  RuntimeStore,
+  withStore,
+} from 'app/utils/stores';
+
+import { FlexRow } from './flex';
+
+const styles = reactStyles({
+  asyncOperationStatusIcon: {
+    width: '.5rem',
+    height: '.5rem',
+    zIndex: 2,
```

Comment:
Most uses of `zIndex` need a comment as to why one needs to override the browser's default layout model. Maybe the usage will make this obvious.

Reply:
Okay, it's not obvious to me why it is needed.

Reply:
I don't know much about how zIndex works (and this retains the existing style's value) but perhaps this is how the circle icon gets superimposed on the thunderstorm icon?